crates/runtime/benches/bench_mssql/tpch_q12.sql

```diff
@@ -0,0 +1,30 @@
+select
+    l_shipmode,
+    sum(case
+            when trim(o_orderpriority) = '1-URGENT'
```

Comment:
The use of `trim()` should be able to be fixed in the arrow converter for MS SQL, by trimming where the source type is `CHAR`. I'll address this in another PR.

TL;DR - `SELECT *` with a `CHAR` in MS SQL returns space-padded values up to the length of the char. However, when MS SQL performs `WHERE` comparisons against `CHAR` columns, it strips the extra padded spaces during the comparison.

This creates problems when we encounter filters that are not pushed down, because the values are returned with padded spaces but the Arrow memory execution doesn't know/expect that.